lib/git_reflow/git_server/bit_bucket/pull_request.rb

```diff
@@ -78,6 +78,24 @@ def approvals
           approved
         end
 
+        def merge_feature_branch(feature_branch_name, options = {})
```

Comment:
Sure, but I wanted to keep the method name the same to be consistent with it's original naming scheme (the original name was `merge_feature_branch`).